src/templates/macros/task_macros.html.twig

```diff
@@ -0,0 +1,47 @@
+{% macro taskListingCurrent(tasksCurrent) %}
+<section class="listing current_week content-flex-item">
+<h2>Current week</h2>
+<table class="tasks-table">
+{% for task in tasksCurrent %}
+<tr>
+<td class="td-width task-drag-and-drop">{{ task.id }}</td>
+<td class="task-name bording-left"><a href="{{ path('app_task_update', {'taskId': task.id} )}}" class="task-link">{{ task.name }}</a></td>
+<td class="td-width bording-left"><a href="{{ path('app_task_delete', {'taskId': task.id} )}}" class="task-link"><i class="fas fa-trash-alt"></i></a></td>
+<td class="td-width bording-left"><a href="" class="task-link"><i class="fa-solid fa-circle-check"></i></a></td>
+</tr>
+{% endfor %}
+</table>
+</section>
+{% endmacro %}
+
+{% macro taskListingNext(tasksNext) %}
```

Comment:
this macro seems to be the same as the above except for one class name - it could/should be merged together